contracts/BSCValidatorSet.sol

```diff
@@ -215,6 +225,42 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   }
 
   /*********************** External Functions **************************/
+  function updateValidatorSetV2() external onlyInit onlyCoinbase onlyZeroGasPrice {
```

Comment:
`updateEligibleValidators` and this `updateValidatorSetV2` can combine to one.

Reply:
Done